src/module/db.ts

```diff
@@ -9,4 +14,53 @@ const DB = new DynamoDB.DocumentClient({
   // endpoint: 'http://localhost:8000',
 });
 
-export default DB;
+export const saveConnection = async (connectionId: string): Promise<DynamoDBResponse<DocumentClient.PutItemOutput>> => {
+  const putParams: DocumentClient.PutItemInput = {
+    TableName: CONNECTIONS_TABLE,
+    Item: {
+      connectionId,
+    },
+  };
+
+  return DB.put(putParams).promise();
+};
+
+export const deleteConnection = async (
+  connectionId: string,
+): Promise<PromiseResult<DocumentClient.DeleteItemOutput, AWSError>> => {
+  const deleteParams: DocumentClient.DeleteItemInput = {
+    TableName: CONNECTIONS_TABLE,
+    Key: {
+      connectionId,
+    },
+  };
+
+  return DB.delete(deleteParams).promise();
+};
+
+export const setUserName = async (
+  connectionId: string,
+  username: string,
+): Promise<DynamoDBResponse<DocumentClient.UpdateItemOutput>> => {
+  const updateParams: DocumentClient.UpdateItemInput = {
+    TableName: CONNECTIONS_TABLE,
+    Key: {
+      connectionId,
+    },
+    ExpressionAttributeNames: { '#usernameKey': 'username' },
+    UpdateExpression: 'set #usernameKey = :usernameVal',
+    ExpressionAttributeValues: {
+      ':usernameVal': username,
+    },
+  };
+
+  return DB.update(updateParams).promise();
+};
+
+export const getAllConnections = async (): Promise<PromiseResult<DocumentClient.ScanOutput, AWSError>> => {
+  const scanParams: DocumentClient.ScanInput = {
+    TableName: CONNECTIONS_TABLE,
+  };
+
+  return DB.scan(scanParams).promise();
+};
```

Comment:
May want to add a filter expression to only get connectionId to possibly save memory space/speed up function if the rows start to get more heavy. 

We may start to add more parameters here (see who is in game and not in game) to limit the number of messages sent. (Something to think about later)

Reply:
will do that later